store/acknowledgement_metrics.go

```diff
@@ -84,7 +127,8 @@ func (m *mongoDB) GetCommunityAvgReportCount(reportType string, meter int, loc s
 	pipeline := []bson.M{
 		aggStageGeoProximity(meter, loc),
 		aggStageReportedToday(todayBeginTime),
-		aggStageUserReportCount(),
+		aggStagePreventNullArray(fields...),
```

Comment:
Just curious, do we need to prevent null array?

Reply:
I still see null values in db.